frappe/utils/background_jobs.py

```diff
@@ -202,7 +202,7 @@ def get_queue(queue, is_async=True):
 	'''Returns a Queue object tied to a redis connection'''
 	validate_queue(queue)
 
-	return Queue(queue, connection=get_redis_conn(), async=is_async)
+	return Queue(queue, connection=get_redis_conn(), is_async=is_async)
```

Comment:
this is pythonrq, we can't change named parameters

Reply:
We can. https://github.com/rq/rq/blob/cc19d1a89eab77ff25522a5b51c258798e620190/rq/queue.py#L61-L72